.github/workflows/build.yml

```diff
@@ -0,0 +1,73 @@
+name: Build
+
+on: [pull_request]
+
+jobs:
+  prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 18.14.0
+          cache: "yarn"
+      - name: Install Yarn v3
+        run: |
+          corepack enable
+          corepack prepare yarn@3.4.1 --activate
```

Comment:
Could we maybe add a script to dev/ or something to run this command if we're switching to yarn? I find myself having to google how to set it up constantly.

Reply:
i added a note in the root README that covers installing yarn v3. it's a 1 and done thing locally. it's linking to the yarn docs, but i could just as easily put the 2 commands directly in the README. WDYT?

Reply:
added helper script in [this commit](https://github.com/xmtp/xmtp-js-content-types/pull/8/commits/d30a2552c4b64a0ec5d4218ad5759b8484ebae0e)